packages/quantic/cypress/integration/result-template/result-template.cypress.ts

```diff
@@ -0,0 +1,33 @@
+import {interceptSearch} from '../../page-objects/search';
+import {ResultTemplateExpectations as Expect} from './result-template-expectations';
+import {ResultTemplateActions as Actions} from './result-template-actions';
+import {configure} from '../../page-objects/configurator';
+
+describe('quantic-result-template', () => {
+  const pageUrl = 's/quantic-result-template';
+  function visitResultTemplate() {
+    interceptSearch();
+    cy.visit(pageUrl);
+    configure({});
+  }
+
+  describe('with default options', () => {
+    it('should work as expected', () => {
+      visitResultTemplate();
+      Expect.displaySlotByName('label', true);
+      Expect.displaySlotByName('badges', true);
+      Expect.displaySlotByName('actions', true);
+      Expect.displaySlotByName('date', true);
+      Expect.displaySlotByName('visual', true);
+      Expect.displaySlotByName('title', true);
+      Expect.displaySlotByName('metadata', true);
+      Expect.displaySlotByName('emphasized', true);
+      Expect.displaySlotByName('excerpt', true);
+      Expect.displaySlotByName('bottom-metadata', true);
+
+      Actions.appendChildren('div', {
```

Comment:
Are you planning on appending stuff to every section? If so then maybe using a map of the slot names and a forEach loop would be a nice way of doing it.

Reply:
I use a enum for that 😆